frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-inventory-card.tsx

```diff
@@ -72,6 +86,13 @@ export const VMInventoryCard: React.FC<VMInventoryCardProps> = () => {
           count={cdromCount}
           TitleComponent={CDROMTitle}
         />
+        <InventoryItem
+          isLoading={isLoading}
+          title="Environment Volume (as Disk)"
```

Comment:
I feel there is too much information in this naming. Maybe you could mention just "Environment Volume" - and it goes to the env tab as well. So we might ignore Environment part when we sort out disks vs cdroms, no?

Reply:
+1, only filtering by type now, removed this part of the code.